frontend/packages/dev-console/src/utils/pipeline-actions.tsx

```diff
@@ -79,8 +79,8 @@ export const triggerPipeline = (
 export const reRunPipelineRun: KebabAction = (kind: K8sKind, pipelineRun: PipelineRun) => ({
   label: 'Rerun',
   callback: () => {
-    const namespace = _.get(pipelineRun, 'spec.metadata.namespace');
-    const pipelineRef = _.get(pipelineRun, 'spec.spec.pipelineRef.name');
+    const namespace = _.get(pipelineRun, 'metadata.namespace');
+    const pipelineRef = _.get(pipelineRun, 'spec.pipelineRef.name');
     if (namespace && pipelineRef) {
       k8sCreate(PipelineRunModel, getPipelineRunData(null, pipelineRun));
     }
```

Comment:
This error does beg the question, why don't we have an error handler here. If we don't have both the namespace & pipelineRef (for whatever reason) we shouldn't just _do nothing_... we should at the bare-minimum show an error dialog.

Reply:
@andrewballantyne What error message should we show in error modal when ```namespace``` or ```pipelineRef``` not found? Do you have something in mind?

Reply:
@andrewballantyne +1, that used to be the flow earlier. IF either namespace or pipelineRef is missing, the function returned logging an error saying "Cannot create pipelinerun because of missing metadata"

Reply:
> @andrewballantyne What error message should we show in error modal when `namespace` or `pipelineRef` not found? Do you have something in mind?

For posterity we discussed this and are waiting on ux feedback
@openshift/team-devconsole-ux @serenamarie125 
See the `#tmp-odc-pipelines` room in Slack for the conversation.

Reply:
> error saying "Cannot create pipelinerun because of missing metadata"

@abhinandan13jan please feel free to add that to the conversation in Slack.

> See the #tmp-odc-pipelines room in Slack for the conversation.

Reply:
yup added

Reply:
Kindly throw the finalized message to the user using the ErrorModal